assembly_image.py

```diff
@@ -0,0 +1,129 @@
+
+
+def get_wardrobe_width(wardrobe):
+    return sum([wardrobe[i]["width"] for i in range(len(wardrobe))])
+
+
+def get_wardrobe_height(wardrobe):
+    return max([wardrobe[i]["height"] for i in range(len(wardrobe))])
+
+
+def get_body_position(wardrobe, body):
+    return sum([wardrobe[i]["width"] for i in range(body)])
+
+
+class Wardrobe:
+    def __init__(self, wardrobe_json):
+        self.body = []
+        self.wardrobe = wardrobe_json
+        self.width = get_wardrobe_width(self.wardrobe)
+        self.height = get_wardrobe_height(self.wardrobe)
+        self.bottom = self.height
+        self.set_bodies_position()
```

Comment:
Puedes hacer que esto sea una propiedad con `@property` o si quieres evitar recalcularla cada vez que la usas con `@cached_property`

```suggestion
        self.bottom = self.height
        self.set_bodies_position()
        
        @cached_property
        def width(self):
            return sum([wardrobe[i]["width"] for i in range(len(wardrobe))])
            
        @cached_property
        def height(self):
            return max([wardrobe[i]["height"] for i in range(len(wardrobe))])
```